build/nsis/setup.nsi

```diff
@@ -10,15 +10,15 @@
   !define NAME "OpenBB Terminal"
   !define COMPANY "OpenBB"
   !define APPFILE "OpenBBTerminal.exe"
-  !define VERSION "1.0.0"
+  !define VERSION "1.2.0"
```

Comment:
Do we want this to be 1.2.1?

Reply:
Next step - we include this version change into the build script